docs/3_development/1_development-flow/3_review.md

```diff
@@ -11,13 +11,13 @@ Document everything, communicate with your team transparently, review together.
 
 - Ensure readability is retained in code e.g. complex nested ternary operators should have a comment note
 - Ensure adequate testing exists where needed
-- PRs include a summary of work which outlines the objectives, decisions & considerations for the PR. Examples:
-    - This PR's objective was to implement middleware for logging, this was completed however presentation & storage of logs is incomplete pending discussion. Certain components are outside of the logging scope at the moment.
+- Code change requests include a summary of work which outlines the objectives, decisions & considerations for the code change. Examples:
+    - This PR's objective was to implement middleware for logging, this was completed however presentation & storage of logs is incomplete pending discussion. Certain components are outside the logging scope at the moment.
     - This PR includes minor styling tweaks on the desktop & mobile navigation menus, refactored navigation list logic.
 
 ## Communication
 
-Ask a team mate on the relevant team channel for a review, this ensures the request is clear as well as gives transparency to the rest of your team if commentary or considerations need to be made.
+Ask a teammate on the relevant team channel for a review, this ensures the request is clear as well as gives transparency to the rest of your team if commentary or considerations need to be made.
```

Comment:
```suggestion
While platforms like Github allow selecting peer reviewers, additionally ask a teammate on the relevant team communication channel for a review. This ensures the request is clear and gives transparency to the rest of your team if commentary or considerations need to be made.
```

Reply:
Actually, makes sense that `Communication` part is under `Requesting reviews`

Reply:
https://github.com/ChainSafe/engineering-handbook/pull/107/commits/2f85bd53bf75c909c33e7c68adb32a20f2d6f5c1